lib/-private/todo-handler.js

```diff
@@ -56,6 +57,12 @@ module.exports = class TodoHandler {
     }
 
     [...existingTodos.values()].forEach((todo) => {
+      let severity = getSeverity(todo);
+
+      if (severity === ERROR_SEVERITY) {
+        return;
+      }
```

Comment:
We _probably_ don't need this short circuit here (since the `result.severity = severity` would just set it back to `ERROR_SEVERITY`), but seems good to avoid the wasted work finding the associated `result` item. :thumbsup:

Reply:
Ya I mainly wanted to early return to avoid unnecessary work.